### PR TITLE
Change return value to return only data field when not empty.

### PR DESCRIPTION
**Pull request by halleyw**
… version. Includes pre-commit fixes.
Updated version to 1.3.1
Change the return type from a call to return data field only when not empty. 
JIRA ticket: https://allenai.atlassian.net/browse/RF-311

### CODE REVIEW

File: erclient/client.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def time_converter(t):
 
         if response and response.ok:
             res_json = response.json()
-            if ('data' in res_json):
+            if ('data' in res_json and len(res_json['data'])):
```
Comment (chrisj-er) [inline erclient/client.py:218]: I think we can bring this in and just say the following. We can rely on the truthiness of whatever data is at res_json["data"]

```
if "data" in res_json and res_json["data"]:
```
Have we seen a case where res_json was None?

Comment (halleyw) [reply erclient/client.py:218]: I have not personally, but it couldn't hurt to add a check for None on line 216

File: erclient/client.py [hunk 2]
```diff
@@ -213,9 +213,9 @@ def time_converter(t):
             response = fn(self._er_url(path), data=body,
                           headers=headers, params=params)
 
-        if response and response.ok:
+        if response and response.ok and response.json() is not None:
```
Comment (chrisj-er) [inline erclient/client.py:216]: It's some well patched code below, so we do need to accept into this if block response and response.ok.

```
if response and response.ok:
    res_json = response.json()
    if res_json and 'data' in res_json and res_json['data']:
        return res_json['data']
    return res_json
```

Comment (halleyw) [reply erclient/client.py]: I see that. Makes sense. I have made the change.

**Review by chrisj-er — APPROVED** [review]

**Pull request merged**